data/templates/secure_erase.py

```diff
@@ -244,12 +242,12 @@ def __run(self):
                     erase_start_flags[index] = os.path.exists(self.path + value + '.log')
             total_percent = sum(percentage_list)/disk_count
             if total_percent == float('+inf'):
-                payload["progress"]["percentage"] = "Not Available"
+                payload["percentage"] = "Not Available"
             else:
-                payload["progress"]["percentage"] = str("%.2f" % total_percent) + "%"
-                payload["progress"]["value"] = int(total_percent)
+                payload["percentage"] = str("%.2f" % total_percent) + "%"
+                payload["value"] = int(total_percent)
```

Comment:
In latest progress design, the property `percentage` will be ignored, any progress need be split into `value` and `maximum`. In your case, you could set a constant `maximum` as `100`.